one-ui/src/test/java/com/marklogic/hub/curation/controllers/FlowControllerTest.java

```diff
@@ -124,8 +125,13 @@ void getFlow() throws IOException {
 
             //POST step
             StepModel stepModel = (StepModel)controller.createStep("testFlow", 1, stepString).getBody();
+            JsonNode flowJson = JSONUtils.convertArtifactToJson(controller.getFlow("testFlow").getBody());
 
             Assertions.assertNotNull(stepModel);
+            //"fileLocations" should not be present in flow declaration
+            Assertions.assertNotNull(flowJson.get("steps").get("1"));
```

Comment:
What happens if an existing flow file has fileLocations in a step, and then saves the flow via QS? Won't the user lose the fileLocations data? This seems risky to me.

Reply:
I just realized this is the one-ui class. I think it's worth adding a comment to the JsonIgnore annotation to define why this is being ignored. Should it even be in the class if it's being ignored??

Reply:
Made changes so that "fileLocations", if present in the step is retained.